src/extension/extension.ts

```diff
@@ -437,7 +445,7 @@ export async function readExtensions(
           join(extensionsDirectory, extensionDir.name),
           extensionDir.name,
         );
-        if (ownedExtensions) {
+        if (ownedExtensions.length > 0) {
```

Comment:
This change is ok, but is it necessary?

```
$ deno
Deno 1.41.0
exit using ctrl+d, ctrl+c, or close()
REPL is running with all permissions allowed.
To specify permissions, run `deno repl` with allow flags.
> let x = []
undefined
> let y = [1,2,3]
undefined
> if (x) { console.log("here"); y.push(...x) }
here
3
> y
[ 1, 2, 3 ]
```

Reply:
It is just that 
````
if (ownedExtensions)
````
will always be true... so I thought it was made for undefined, but it never is. That is what I added the length condition. 

But indeed `push(...[])` is ok. So maybe no if is needed there... 

unless it is there for other reason and I need to learn something new.